prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md

```diff
@@ -0,0 +1,39 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What tags are necessary for a complete HTML Skeleton?
```

Comment:
make sure to answer all of these quiz questions

Reply:
Quiz questions added, thank you! 

![Asnwers_added](https://user-images.githubusercontent.com/91649194/222031507-54ad7ca0-e223-4648-a348-569f97b353f1.PNG)